frontend-service/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "eea-reportnet-frontend",
-  "version": "3.0.0",
```

Comment:
We don't change this version number, this is not the microservice version.

The version from all microservices are in parent-poms/parent/pom.xml.

Reply:
Thanks @edual16 good to know! So the FE doesn't follow any versioning in the package.json? 
Either way this is intended just to get used to the process. I will do some other dummy change and come back to you.

Reply:
One example: the bulkDeleteCheckTooltipShift message could be changed from
Hold Shift+Click to select more than one field
to 
Hold shift to select more than one field
Or something similar

Reply:
Done @alfaredu please take a look at my comments on what I've done. I'd like to have a short call if you want at some point about the application overview. Will ping you in Teams. Many thanks!